pkgs/applications/graphics/qcustomplot/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchurl, qt5, gcc, cmake, pkg-config }:
+stdenv.mkDerivation rec {
+  pname = "qcustomplot";
+  version = "2.1.1";
+  src = fetchurl {
+    url = "https://www.qcustomplot.com/release/${version}/QCustomPlot-source.tar.gz";
+    sha256 = "sha256:Xi0i3sd5248B81fL2yXlT7z5ca2u516ujXrSRESHGC8=";
+  };
+  patches = [ ./add-cmake.patch ];
+  cmakeFlags = [ "-DUSE_QT_VERSION=5" ];
```

Comment:
I have found that Archlinux does not use cmake, but qmake.

https://gitlab.archlinux.org/archlinux/packaging/packages/qcustomplot/-/blob/main/PKGBUILD?ref_type=heads

Remove the cmake patches - let's respect upstream.